pkg/ee/mutation/resourcequota/mutation.go

```diff
@@ -117,6 +118,8 @@ func ensureProjectOwnershipRef(ctx context.Context, client ctrlruntimeclient.Cli
 	ownRefs = append(ownRefs, projectRef)
 	resourceQuota.SetOwnerReferences(ownRefs)
 
+	kubernetes.EnsureLabels(resourceQuota, map[string]string{kubermaticv1.ResourceQuotaSubjectHumanReadableNameLabelKey: project.Spec.Name})
```

Comment:
Can you handle labels from `ensureResourceQuotaLabels` in the same way and delete that function as part of refactoring in this PR? Or you prefer to do it separately?

Reply:
good point, ill change it